### PR TITLE
Translation nl

### PR DESCRIPTION
**Pull request by plunched**
added some translations to dutch | nl

### CODE REVIEW

**Comment by plunched** [comment]
gotta resolve the conflicts I guess

**Comment by plunched** [comment]
will do more later on

**Review by overlisted — APPROVED** [review]

**Review by samarmohan — APPROVED** [review]

**Comment by amitojsingh366** [comment]
please follow the proper pattern for commit messages *(💡TIP: you can use `yarn commit` or `yarn commit:signed` and answer the questions asked to generate a properly formatted one for you!)*

to fix your most recent commit you can run the command `git commit --amend -m "properly formatted commit message here"`

to fix your older/multiple commits please take a look at [this article](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message#amending-older-or-multiple-commit-messages)

File: kibbeh/public/locales/nl/translation.json [hunk 1]
```diff
@@ -15,15 +15,15 @@
     "copied": "Gekopieerd",
     "formattedIntlDate": "{{date, intlDate}}",
     "formattedIntlTime": "{{time, intlTime}}",
-    "requestPermissions": "Houd er rekening mee dat het uitvoeren van DogeHouse zonder de benodigde permissies ongewenste fouten kan veroorzaken"
+    "requestPermissions": "Hou er rekening mee dat het uitvoeren van DogeHouse zonder de benodigde permissies ongewenste fouten kan veroorzaken"
```
Comment (laurens-vrh) [inline kibbeh/public/locales/nl/translation.json:18]: The old line is more formal Dutch

File: kibbeh/public/locales/nl/translation.json [hunk 2]
```diff
@@ -85,16 +85,15 @@
     "myProfile": {
       "logout": "Afmelden",
       "probablyLoading": "Waarschijnlijk aan het laden...",
-      "voiceSettings": "Ga naar spraakinstellingen",
-      "soundSettings": "Ga naar geluidsinstellingen",
-      "deleteAccount": "Verwijder account",
-      "overlaySettings": "Ga naar overlay-instellingen",
-      "couldNotFindUser": "Sorry, die gebruiker kon niet gevonden worden",
-      "privacySettings": "Privacy settings"
```
Comment (laurens-vrh) [inline kibbeh/public/locales/nl/translation.json:93]: Why is the privacy settings translation gone?

File: kibbeh/public/locales/nl/translation.json [hunk 3]
```diff
@@ -85,16 +85,15 @@
...
+      "voiceSettings": "Ga naar spraak-instellingen.",
+      "soundSettings": "Ga naar geluids-instellingen.",
+      "deleteAccount": "Verwijder account.",
+      "overlaySettings": "Ga naar overlay-instellingen.",
+      "couldNotFindUser": "Sorry, die gebruiker kon niet gevonden worden."
     },
     "notFound": {
       "whoopsError": "Oeps! Deze pagina is verloren gegaan tijdens het gesprek.",
-      "goHomeMessage": "Maak je geen zorgen. Je kan",
+      "goHomeMessage": "Maak je geen zorgen. Je kan naar de homePagina gaan",
```
Comment (laurens-vrh) [inline kibbeh/public/locales/nl/translation.json:96]: home-pagina*

File: kibbeh/public/locales/nl/translation.json [hunk 4]
```diff
@@ -15,15 +15,15 @@
...
-    "requestPermissions": "Houd er rekening mee dat het uitvoeren van DogeHouse zonder de benodigde permissies ongewenste fouten kan veroorzaken"
+    "requestPermissions": "Hou er rekening mee dat het uitvoeren van DogeHouse zonder de benodigde permissies ongewenste fouten kan veroorzaken"
   },
   "header": {
     "_comment": "Main Header UI Internationalization Strings",
     "title": "DogeHouse",
-    "mutedTitle": "Gedempt | DogeHouse",
-    "deafenedTitle": "Niet hoorbaar | DogeHouse",
+    "mutedTitle": "Gemute | DogeHouse",
+    "deafenedTitle": "dedempt | DogeHouse",
```
Comment (laurens-vrh) [inline kibbeh/public/locales/nl/translation.json:24]: Typing error

**Review by laurens-vrh — CHANGES_REQUESTED** [review]
Maybe change some things

**Comment by amitojsingh366** [comment]
stale pr, gonna close

**Pull request closed without merging**